src/components/LineChart.vue

```diff
@@ -47,168 +331,257 @@ export default {
-                            .y(function(d) { return self.yScale(d.n)})
-                            (d.values)
-                    });
-                // .merge()
-                // .transition()
-                // .duration(this.duration);
+        //     // Update Line
+        //     lineGroup.selectAll(".line")
+        //         .data(groupedData)
+        //         .enter()
+        //         .append("path")
+        //             .attr("class","line")
+        //             .attr("fill", "none")
+        //             .attr("stroke", "coral")
+        //             .attr("stroke-width", 1) 
+        //             .attr("d", function(d){
+        //                 return d3.line()
+        //                     // .curve(this.d3.curveMonotoneX)
+        //                     .x(function(d) { return self.xScale(d.dateObj)})
+        //                     .y(function(d) { return self.yScale(d.n)})
+        //                     (d.values)
+        //             });
+        //         // .merge()
+        //         // .transition()
+        //         // .duration(this.duration);
                 
-            // add annotation
+        //     // add annotation
             
-            // find date of peak swe
+        //     // find date of peak swe
            
-            // let peakSWE = filteredData.reduce((acc, day) => {
-            //     return acc + day.SWEin;
-            // });
-            let peakSWE = Math.max.apply(Math.filteredData.map(function(o){return o.SWEin;}));
-            console.log(peakSWE,"Peak")
+        //     // let peakSWE = filteredData.reduce((acc, day) => {
+        //     //     return acc + day.SWEin;
+        //     // });
+        //     let peakSWE = Math.max.apply(Math.filteredData.map(function(o){return o.SWEin;}));
+        //     console.log(peakSWE,"Peak")
             
 
-        }
+        // }
     }
 }
 </script>
 
-<style scoped>
-    #swe-chart {
-        /* background-color: coral; */
+<style lang="scss" scoped>
+    svg {
+        width: 800px;
+        height: 500px;
     }
 
-    .line {
-        stroke: black;
+   .axis-tick, .annotation-line {
+        fill: none;
+        stroke: #d9d9d9;
+        stroke-width: 0.75px;
+        stroke-linejoin: round;
+      }
+
+    .axis-label {
+        font-size: 12px;
+        font-weight: 300;
+    }
+
+    .axis-title {
+        font-family: SourceSansPro-Black, Source Sans Pro;
+        font-weight: 800;
+        fill: #fff;
+        font-size: 26px;
+    }
+    .reveal{
+        display: none;
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        right: 0;
+        width: 100%;
+        background: white;
+        animation: slide 5s cubic-bezier(.5,.5,0,1);
+        animation-fill-mode: forwards;
+        animation-delay: 5s;
+        // animation-delay: 5s;
+        rect {
+            fill: white;
+        }
+       
+    }
+
```

Comment:
I was going to say that you could still animate these lines without drawing them in d3 but looks like you're on it 👍🏼